index.html

```diff
@@ -19,13 +25,15 @@
         <div class="form-field">
           <label for="level" class="form-label">Level*</label>
           <select name="level" class="form-input">
+            id = "level"
             <option value="basic">Basic</option>
             <option value="advanced">Advanced</option>
           </select>
         </div>
         <div class="form-field">
           <label for="hoursPerWeek" class="form-label">Hours per week*</label>
           <input
+            id="hoursPerWeek"
```

Comment:
Adding id attributes to form elements enhances accessibility and makes it easier to manipulate these elements with JavaScript.